wordpress-mobile/WordPress-Android
# Fixes: Context menu icons not shown in Pages item

## Fixes
https://github.com/wordpress-mobile/WordPress-Android/issues/19347

## Description
This PR fixes the context menu icons not being shown 

📔 Note: The icons and the context menu design will be updated in a seperate PR

## Screenshot
<img src="https://github.com/wordpress-mobile/WordPress-Android/assets/17463767/97ed36a6-40fc-4d37-a683-e43d272c486f" width="250" height="580">

## To test:
- Login to Jetpack app 
- Go to Pages 
- Click on Context menu
- Verify that the context menu icon is shown 


## Regression Notes
1. Potential unintended areas of impact
Context icon is not shown

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:

UI Changes testing checklist:
